.gitignore

```diff
@@ -3,3 +3,5 @@ bin/
 .project
 .classpath
 .settings/
+*.iml
+.idea/
```

Comment:
Side note: Are you using InteliJ? Would be cool to add a little note to the contribution guide how to setup tycho with InteliJ then (as a separate PR)!

Reply:
yes I'm using InteliJ. Frankly nothing special/magic I just imported the project as a Maven project.
Not sure if that worth a PR. Bear in mind I'm a Maven cli guy so not running test via IDE :)